OSSM-6063 [DOC] OpenShift Service Mesh Console (OSSMC) plugin Integration (#88446)
openshift/openshift-docs: modules/ossm-about-console-plugin.adoc

Traffic:: Shows information about all inbound and outbound traffic to the workload.

Logs:: Shows the logs for the workload's containers. You can see container logs individually ordered by log time and how the Envoy sidecar proxy logs relate to your workload's application logs. You can enable the tracing span integration, which allows you to see which logs correspond to trace spans.

Metrics:: Shows inbound and outbound metric graphs in the corresponding subtabs. All the workload metrics are here, providing a detailed view of the performance of your workload. You can enable the tracing span integration, which allows you to see which spans occurred at the same time as the metrics. With the span marker in the graph, you can see the specific spans associated with that timeframe.

Traces:: Provides a chart showing the trace spans collected over the given timeframe. The trace spans show the most low-level detail within your workload application. The trace details further show heatmaps that provide a comparison of one span in relation to other requests and spans in the same timeframe.

Envoy:: Shows information about the Envoy sidecar configuration.

> In the web console *Networking* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab similar to the *Workloads* details page. In the web console *Projects* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab that provides traffic graph information about that project. It is the same information shown in the *Traffic Graph* page but specific to that project.

Reviewer: Networking and Projects are different pages with different features. I think they should be in different paragraphs. If this does not fit with doc guidelines we can rephrase the statements.

```suggestion
In the web console *Networking* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab similar to the *Workloads* details page. 

In the web console *Projects* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab that provides traffic graph information about that project. It is the same information shown in the *Traffic Graph* page but specific to that project.
```

// Module included in the following assemblies:
//
// * service-mesh-docs-main/observability/kiali/ossm-console-plugin.adoc

:_mod-docs-content-type: CONCEPT
[id="ossm-about-console-plugin_{context}"]
= About {sm-plugin-full}

> The {SMPlugin} is an extension to {ocp-short-name} web console that provides visibility into your {SMProductShortName}.

Reviewer 2: ```suggestion
The {SMPlugin} is an extension to {ocp-product-title} web console that provides visibility into your {SMProductShortName}.
```

// Module included in the following assemblies:
//
// * service-mesh-docs-main/observability/kiali/ossm-console-plugin-assembly.adoc

:_mod-docs-content-type: CONCEPT
[id="ossm-about-console-plugin_{context}"]
= About {SMPlugin-s}

> The {SMPlugin} is an extension to {ocp-short-name} web console that provides visibility in your {SMProductShortName}.

Reviewer 3: maybe, "visibility into your"

// Module included in the following assemblies:
//
// * service-mesh-docs-main/observability/kiali/ossm-console-plugin.adoc

:_mod-docs-content-type: CONCEPT
[id="ossm-about-console-plugin_{context}"]
= About {sm-plugin-full}

The {SMPlugin} is an extension to {ocp-short-name} web console that provides visibility into your {SMProductShortName}.

> You can install the {SMPluginShort} with Kiali Operator by craeting a `OSSMConsole` resource with the corresponding plugin settings.

Reviewer: I don't know how this is done for other RH products, but I think the compatibility matrix should be in the installation section. It indicates in which OCP versions you can install the OSSMC plugin.

Traffic:: Shows information about all inbound and outbound traffic to the workload.

Logs:: Shows the logs for the workload's containers. You can see container logs individually ordered by log time and how the Envoy sidecar proxy logs relate to your workload's application logs. You can enable the tracing span integration, which allows you to see which logs correspond to trace spans.

Metrics:: Shows inbound and outbound metric graphs in the corresponding subtabs. All the workload metrics are here, providing a detailed view of the performance of your workload. You can enable the tracing span integration, which allows you to see which spans occurred at the same time as the metrics. With the span marker in the graph, you can see the specific spans associated with that timeframe.

Traces:: Provides a chart showing the trace spans collected over the given timeframe. The trace spans show the most low-level detail within your workload application. The trace details further show heatmaps that provide a comparison of one span in relation to other requests and spans in the same timeframe.

Envoy:: Shows information about the Envoy sidecar configuration.

> In the web console *Networking* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab similar to the *Workloads* details page.

Reviewer: I suggest some small new content that I forgot to add the other day.

```suggestion
In the web console *Networking* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab similar to the *Workloads* details page.

In the web console *Projects* details page, the {SMPluginShort} adds a *{SMProductShortName}* tab that provides traffic graph information about that project. It is the same information shown in the *Traffic Graph* page but specific to that project.
```
